test/browser/test/userEvent.test.ts

```diff
@@ -135,9 +135,10 @@ describe('userEvent.click', () => {
       },
     })
 
+    // not exact due to scaling and rounding
     expect(spy).toHaveBeenCalledWith({
-      x: 200,
-      y: 150,
+      x: expect.closeTo(200, -1),
+      y: expect.closeTo(150, -1),
```

Comment:
Scaling round issue seems to already exist for UI case. I can reproduce it on main branch with firefox.

```ts
$ PROVIDER=playwright BROWSER=firefox pnpm -C test/browser test-fixtures /userEvent. -t 'x/y'

 FAIL  test/userEvent.test.ts > userEvent.click > clicks with x/y coords
AssertionError: expected "spy" to be called with arguments: [ { x: 200, y: 150 } ]

Received: 

  1st spy call:

  Array [
    Object {
-     "x": 200,
+     "x": 199,
      "y": 150,
    },
  ]
```